pages/common/r.md

```diff
@@ -0,0 +1,15 @@
+# R
```

Comment:
It really is a single letter command that's uppercase? That's rather unusual.

Reply:
Yup! Though I've seen many versions where it calls for the lower case, though that may just be aliased.

Reply:
I would really hate to break convention here. All our pages have the heading as lowercase. Unless there is a different command with a lower case `r`, I would recommend changing this to lowercase.